tensorboard/plugins/projector/tf_projector_plugin/index.js

```diff
@@ -0,0 +1,20 @@
+export function render() {
+  const style = document.createElement('style');
+  style.innerText = `
+html,
+body {
+  height: 100%;
+  margin: 0;
+  width: 100%;
+}`;
+  document.head.appendChild(style);
+
+  const iframe = document.createElement('iframe');
+  iframe.src = './projector_binary.html';
+  Object.assign(iframe.style, {
```

Comment:
If you like, you could just use

```css
html,
iframe,
body {
  border: 0;
  height: 100%;
  margin: 0;
  width: 100%;
}
```

on the stylesheet to slim this down a bit. But no action required if you
prefer it this way.